### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by evgenius-kom**

### CODE REVIEW

File: GameLogic/entities/pieces/PieceConstants.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ static class PieceConstants
         public static readonly ISet<int> InvalidPawnRanks = new HashSet<int> { Y._0, Y._7 };
 
         public static readonly ISet<PieceType> PossiblePromotionPieceTypes =
-            new HashSet<PieceType>{PieceType.Knight, PieceType.Bishop, PieceType.Rook, PieceType.Queen};
+            new HashSet<PieceType> { PieceType.Knight, PieceType.Bishop, PieceType.Rook, PieceType.Queen };
```
Comment (SherlockKA) [inline GameLogic/entities/pieces/PieceConstants.cs:18]: Are spaces correct? As I get it, you planned to undo adding such spaces.

File: GameLogic/engine/StandardFENSerializer.cs [hunk 2]
```diff
@@ -151,7 +151,11 @@ public static string BoardToNotation(IRectangularBoard board)
                     // Set |numberEmptyCells| to zero.
                     else
                     {
-                        if (numberEmptyCells != 0) row.Add(Convert.ToChar(numberEmptyCells + 48));
+                        if (numberEmptyCells != 0)
```
Comment (SherlockKA) [inline GameLogic/engine/StandardFENSerializer.cs:154]: Out of curiosity, did you change it manually? Or did 'dotnet format' edit it?

**Review by SherlockKA — APPROVED** [review]

File: GameLogic/entities/pieces/PieceConstants.cs (hunk 1 above)
Comment (evgenius-kom) [reply GameLogic/entities/pieces/PieceConstants.cs:18]: These spaces are not the same as I showed recently

File: GameLogic/engine/StandardFENSerializer.cs (hunk 2 above)
Comment (evgenius-kom) [reply GameLogic/engine/StandardFENSerializer.cs:154]: .editorconfig, yes
I prefer to highlight blocks like this

**Pull request merged**